first_hour_on_vwb/working_with_data_collections.ipynb

```diff
@@ -0,0 +1,603 @@
+    "Build upon the best practices demonstrated in this notebook to create and share your own data collections. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "d975e698c9a4"
+   },
+   "source": [
+    "### Objective\n",
+    "\n",
+    "This notebook will guide you through creating a new data collection or converting an existing workspace into a data collection which you can then share with collaborators and use in your workspaces.\n",
+    "\n",
+    "- [Create a new data collection](#create-new-dc)\n",
+    "    - [Create a new workspace.](#create-new-ws)\n",
+    "    - [Convert the workspace into a data collection](#convert-to-dc)\n",
+    "    - [Publish an initial version](#publish-version)\n",
+    "    - [Add referenced and controlled resources to your new workspace](#add-resources)\n",
+    "- [Add a data collection to your workspace](#add-dc-to-ws)\n",
+    "\n",
+    "#### How to run this notebook\n",
+    "\n",
+    "Please run the [Setup](#setup) section before running any other section in this notebook.\n",
+    "\n",
+    "#### Costs\n",
+    "\n",
+    "This notebook takes less than a minute to run, which will typically cost less than $0.01 of compute time on your cloud environment."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Setup\n",
+    "<a id=\"setup\"></a>\n",
+    "\n",
+    "Run the cell below to capture the ID of the current workspace. You'll use this value to return to the current workspace after you've created a new workspace as part of the process of creating a data collection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import ipywidgets as widgets\n",
+    "import subprocess\n",
+    "import widget_utils as wu\n",
+    "import vwb_folder_utils as vfu\n",
+    "from datetime import date\n",
+    "\n",
+    "'''\n",
+    "Resolves ID of current workspace.\n",
+    "'''\n",
+    "def get_current_workspace_id():\n",
+    "    CURRENT_WORKSPACE_ID_CMD_OUTPUT = !terra workspace describe --format=json | jq --raw-output \".id\"\n",
+    "    CURRENT_WORKSPACE_ID = CURRENT_WORKSPACE_ID_CMD_OUTPUT[0]\n",
+    "    return CURRENT_WORKSPACE_ID\n",
+    "\n",
+    "CURRENT_WORKSPACE_ID = get_current_workspace_id()\n",
+    "print(f\"Current workspace ID is {CURRENT_WORKSPACE_ID}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Create a new data collection\n",
+    "<a id=\"create-new-dc\"></a>\n",
+    "\n",
+    "Before creating your data collection, consider the following:\n",
+    "1. What data do you want to share? What type of resources--Cloud Storage buckets or objects, BigQuery tables, GitHub repositories--will be made available via this data collection?\n",
```

Comment:
```suggestion
    "1. What data do you want to share? What type of resources--Cloud Storage buckets or objects, BigQuery tables--will be made available via this data collection?\n",
```
I think based on some older implementation, github repos were able to be a part of data collections. I don't _think_ thats the case anymore, and I don't think we intend to have repos be part of a data collection.

Reply:
Harris confirmed that this is not the case any more.

Reply:
Sorry @amygdala can you please clarify what is not the case? Is the statement "Github repos are not a part of data collections" true?

Reply:
sorry!  your statement is true: github repos can no longer be part of data collections.
(this changed with the version support)

Reply:
fixed!